turtleFSI/modules/newtonsolver.py

```diff
@@ -103,8 +104,8 @@ def newtonsolver(F, J_nonlinear, A_pre, A, b, bcs, lmbda, recompute, recompute_t
         # Check residual
         residual = b.norm('l2')
         rel_res = norm(dvp_res, 'l2')
-        if rel_res > 1E20 or residual > 1E20:
-            raise RuntimeError("Error: The simulation has diverged during the Newton solve.")
+        if rel_res > 1E20 or residual > 1E20 or rel_res is np_nan or residual is np_nan:
```

Comment:
Is it correct to use `is` in this case? Or should it be `==` or `np.isnan`?

Reply:
Good point! I think it should be `np.isnan`